pkg/performanceprofile/controller/performanceprofile/components/utils.go

```diff
@@ -103,9 +116,11 @@ func NewCPULists(reservedList, isolatedList string) (*CPULists, error) {
 	if err != nil {
 		return nil, err
 	}
+	offlined, _ := cpuset.Parse(offlinedList)
```

Comment:
why are we ignoring the error here?

Reply:
This also triggers a bell. In a future PR, we should make sure we get and handle the parsing errors for the cpu sets above.

Reply:
missed. fix.

Reply:
It seems to me we are still ignoring the error? should be
```
offlined, err := cpuset.Parse(offlinedList)
if err != nil {
   ...
```